src/commands/interactive/inputInteractiveDate.ts

```diff
@@ -0,0 +1,20 @@
+import { Input } from "../../deps.ts";
+import { CmdGlobalOptions } from "../cmd-options.ts";
+import { dateRegex } from "../custom-types.ts";
+
+export async function interactiveDate(
+  _options: CmdGlobalOptions,
+  question: string,
+) {
+  let running = true;
+  let date = "";
+  while (running) {
+    date = await Input.prompt(question);
+    if (!dateRegex.test(date.toLowerCase())) {
```

Comment:
t: we could maybe also allow that the user types "quit" or "QUIT" if he/she wishes to exit, just as an idea

Reply:
Good idea. I also added a "back" function, just in case someone needs that.